Include/cpython/code.h

```diff
@@ -199,6 +199,9 @@ struct PyCodeObject _PyCode_DEF(1);
 */
 #define CO_HAS_DOCSTRING 0x4000000
 
+/* A function defined in class scope */
+#define CO_METHOD  0x8000000
```

Comment:
I'm not sure it matters, but you could use a smaller constant like `0x0040` (between `CO_GENERATOR` and `CO_COROUTINE`)